src/Commands/InteractsWithElasticsearchCommand.php

```diff
@@ -45,4 +47,27 @@ public function switchAlias(string $alias, string $index): void
             }
         }
     }
+
+    public function deleteIndex(string $index): void
+    {
+        $this->elasticsearch->indices()->delete(['index' => $index]);
+    }
+
+    public function getStores()
+    {
+        $storeModel = config('rapidez.models.store');
+        return $this->argument('store')
+            ? $storeModel::where('store_id', $this->argument('store'))->get()
+            : $storeModel::all();
+    }
+
+    public function setStore(Store $store): void
+    {
+        config()->set('rapidez.store', $store->store_id);
+        config()->set('rapidez.website', $store->website_id);
+        $code = config('rapidez.models.store')::getCachedWhere(function ($store) {
+            return $store['store_id'] == config('rapidez.store');
+        })['code'];
+        config()->set('rapidez.store_code', $code);
+    }
```

Comment:
Do keep in mind that these functions do not do any interaction with Elasticsearch 😉 
Maybe this logic could become a trait, or some of this logic could be moved to a Facade(/The Rapidez facade)